website/app/styles/pages/application/header.scss

```diff
@@ -139,75 +139,36 @@
   }
 }
 
-.doc-page-header__nav-item-generic {
-  a {
-    display: block;
-    color: var(--doc-color-gray-400);
-
-    &:visited {
-      color: var(--doc-color-gray-400);
-    }
-
-    &:hover {
-      color: var(--doc-color-white);
-    }
-
-    &:focus-visible {
-      border-radius: 3px;
-      outline: 2px solid var(--doc-focus-ring-color);
-    }
-  }
-}
-
 .doc-page-header__nav-item--split {
   margin: 0 0 0 auto;
 }
 
-// "fake" search button (this triggers another button, the "Algolia search" one, which is hidden)
-
-.doc-page-header__search-button {
+.doc-page-header__desktop-icon-item {
   @include doc-font-style-navigation();
-  @include doc-header-custom-focus ();
-  display: flex;
-  gap: 4px;
-  align-items: center;
-  height: 36px;
-  margin: 0;
-  padding: 4px 12px;
+  @include doc-header-custom-focus();
+  display: block;
+  margin: 0 12px;
+  padding: 0;
   color: var(--doc-color-gray-400);
-  background-color: var(--doc-color-gray-100);
-  border: 1px solid var(--doc-color-gray-300);
-  border-radius: 3px;
+  background-color: transparent;
+  border: 1px solid transparent;
   cursor: pointer;
 
-  &:hover {
-    color: var(--doc-color-white);
-  }
-
-  &:focus {
-    border-color: var(--doc-focus-ring-color);
-    outline: 1px solid var(--doc-focus-ring);
-    outline-offset: 0;
-  }
-
   svg {
-    display: block;
-    width: 16px;
-    height: 16px;
-    color: inherit;
+    width: 24px;
+    height: 24px;
```

Comment:
It makes sense that the icon sizes were increased to 24px because they need the larger click target size, but they were only increased via the CSS. The icon size must also change from 16px to 24px to get the proper stroke weight.